core/src/main/scala/org/dbpedia/extraction/config/dataparser/InfoboxMappingsExtractorConfig.scala

```diff
@@ -0,0 +1,18 @@
+package org.dbpedia.extraction.config.dataparser
+
+/**
+  * Created by aditya on 6/4/16.
+  */
+object InfoboxMappingsExtractorConfig {
+
+  val infoboxNameMap = Map(
+    "en" -> "Infobox",
+    "no" -> "Infoboks"
+
+  )
+
+  val KeyWordsMapForP856 = Map(
```

Comment:
I think this would more extensible if we changed it to 
```
val directTemplateMapsToWikidata = Map(
    "en" -> Map("Official website" -> "856", "Official URL" -> "856"),
    "no" -> Map("BetingetURL" -> "856")
  )
```